packages/odyssey-react/src/components/TextInput/TextInput.module.scss

```diff
@@ -87,6 +87,23 @@
   padding-inline-start: var(--PaddingInline);
 }
 
+.affixHidden {
+  display: none;
```

Comment:
Missed this on my first pass -- `opacity: 0` would be better here (with `opacity: 1` for the reveal below). That way, an assistive device will still pick up the `.affix` content. 

It'll also help keep the Input contents from jumping around since it will take up space before it's visually revealed.